### ISSUE TITLE
Find a better solution for using images built in PRs in deploy-kfp-tekton.sh and deploy-kfp.sh

### ISSUE THREAD

**Issue by hbelmiro** [kubeflow/pipelines#11293]
Currently [deploy-kfp-tekton.sh](https://github.com/kubeflow/pipelines/blob/4ccb047771f6dae5d612d176d0e23c2fbb97e6a1/scripts/deploy/github/deploy-kfp-tekton.sh) and [deploy-kfp.sh](https://github.com/kubeflow/pipelines/blob/4ccb047771f6dae5d612d176d0e23c2fbb97e6a1/scripts/deploy/github/deploy-kfp.sh) [are patching the deployments to set the images](https://github.com/kubeflow/pipelines/pull/11284). We need to find a better solution. Maybe using [kustomize](https://kustomize.io)?

This issue depends on:

- https://github.com/kubeflow/pipelines/issues/11239
- https://github.com/kubeflow/pipelines/issues/11292

### Labels

/area testing

---

Impacted by this bug? Give it a 👍.

**Comment by CarterFendley** [comment]
What's up homies.

I opened #11294 to add this for Argo setups. It looks like the tekton action [already uses](https://github.com/kubeflow/pipelines/blob/master/scripts/deploy/github/deploy-kfp-tekton.sh#L36) a [kustomization.yaml](https://github.com/kubeflow/pipelines/blob/4ccb047771f6dae5d612d176d0e23c2fbb97e6a1/scripts/deploy/github/manifests/kustomization.yaml) for its' setup.

This file has configurations for the `apiserver`, `persistenceagent`, and `scheduledworkflow` but no patches for the driver / launcher. Since all of this stuff is the same, **I am wondering if I should try to pull out these images configurations into a common file**?

I am not 100% certain on the tekton side of things, so please add info where helpful.

**Comment by hbelmiro** [comment]
I believe both ArgoWF and Tekton use the same images.
@rimolive can you confirm it?

**Comment by rimolive** [comment]
Not really. In KFP 2.2.0 we did not finish the kfp-tekton code merge, so if you need to use tekton you will still require a separate image.

**Comment by droctothorpe** [comment]
After reviewing this issue, we've decided to close it:

No longer relevant.

If you believe this was closed in error, please feel free to reopen with additional context.
